### ISSUE TITLE
Collect design ideas and gameplay inspiration

### ISSUE THREAD

**Issue by t-karcher** [ChrisOKay/game-off-2016#3]
I'll leave this here so I don't forget to mention it in the next meeting:
[Dragon's Crown example level](https://youtu.be/ostW_NDRc2s?t=20m37s)
What I like:
- 2D side scrolling adventure incl. chests and secret passages 
- lots of parallax layers (more than five layers, some in front, some in the back)
- cut scenes at the end of the levels for story telling (e.g. a defeated boss revealing the location of the next light?)

**Comment by t-karcher** [comment]
By the way: We're not limited to the ground! Some scenes could also be located on a roof, or in the subway.

**Comment by t-karcher** [comment]
Another inspiration: Limbo.
https://youtu.be/PE-U1k4XUQk

**Comment by ChrisOKay** [comment]
warst du bei der letzten Diskussion kurz eingenickt? ;)

**Comment by t-karcher** [comment]
;-p Ich hatte nur heute schon fast wieder den Namen des Spiels vergessen. Deshalb hier nochmal für mich zur Erinnerung.

**Comment by t-karcher** [comment]
Music inspiration: https://www.youtube.com/watch?v=VzMh1O-MJYE

**Comment by ChrisOKay** [comment]
Puzzle-Idee: Farbfilter: semitransparente Farbwand "vor" dem Spieler durch scrollen auf mehr oder weniger wilden Farbmuster auf Wand / Hintergrund, um Information/Schalter/Tür sichtbar zu machen

**Comment by t-karcher** [comment]
Hinweise auf Zahlen und Code-Wörter könnten durch farbig blinkende Lichter an den Spieler übermittelt werden:

![stranger-things-on-netflix](https://cloud.githubusercontent.com/assets/22897030/20041915/0792d2e0-a471-11e6-837a-db555f53c48a.jpg)

**Comment by t-karcher** [comment]
Licht-Puzzle in verschiedenen Schwierigkeitsgraden: https://www.khanacademy.org/math/math-for-fun-and-glory/puzzles/lights-puzzles/e/lights-puzzles-1

**Comment by ChrisOKay** [comment]
12 Monkeys:

<img width="700" alt="bildschirmfoto 2016-11-10 um 05 57 35" src="https://cloud.githubusercontent.com/assets/22896985/20165101/b1ad6808-a70a-11e6-97dd-be9f6461bdcd.png">

**Comment by t-karcher** [comment]
Gefällt mir! Das Spiel entwickelt sich damit zwar mehr in Richtung Point-and-Click-Adventure, aber das ist ja nichts Schlechtes. Dann steht eben "Hacking" (=Puzzles lösen) im Vordergrund.

Um die grafische Gestaltung zu erleichtern (damit nicht jedes Detail / jeder Knopf in die Landschaft eingebaut werden muss), werd' ich mal versuchen, das obige Rätsel in einer Art "Pop-Up" a la Tiny Bang Story zu präsentieren (siehe unten). Oder was meint ihr?

![the-tiny-bang-story-33](https://cloud.githubusercontent.com/assets/22897030/20173751/edd4e04c-a73a-11e6-9809-b41318d3fe64.jpg)

**Comment by ChrisOKay** [comment]
finde den mix zwischen maus und tastatur zu springen erstmal zu hart, speziell wenn nur wenige rätsel so zu lösen wären. aber man könnte das panel ja dann alternativ über die bewegungstasten bedienen. eine live-aktualisierung der lichter, damit man sieht was man macht, wäre gut.

**Comment by t-karcher** [comment]
Ja, daran hatte ich auch schon gedacht. Wie wäre es, ganz auf die Tastatur zu verzichten und komplett auf Maus-Steuerung zu gehen? Dann könnte man das Spiel später vielleicht sogar auf einem Tablet zum Laufen kriegen...

**Comment by ChrisOKay** [comment]
Wir könnten dieses "simon-sagt" farb-drück spiel integrieren, um türen zu öffnen. Um den Fortschritt damit etwas zu steuern, muss man dazu erst die entsprechenden "farben" der stadt freigeschaltet haben.

**Comment by t-karcher** [comment]
Noch eine Quelle der Inspiration, was das Design angeht: "Vive le Roi" https://www.youtube.com/watch?v=1TuB9K4quuU
